distributed/scheduler.py

```diff
@@ -2736,6 +2745,11 @@ def transition_erred_released(self, key):
                 if dts._state == "erred":
                     recommendations[dts._key] = "waiting"
 
+            w_msg = {"op": "free-keys", "keys": [key], "reason": "Erred->Released"}
+            for w in ts._erred_on:
+                worker_msgs[w] = [w_msg]
```

Comment:
Minor nit, we've been pretty consistent around the pronoun `ws` for WorkerState objects.

```suggestion
            for ws in ts._erred_on:
                worker_msgs[ws] = [w_msg]
```